asf_online_data_exploration/utils/data_collection_utils.py

```diff
@@ -0,0 +1,37 @@
+"""
+Utility functions for data collection.
+"""
+
+import json
+import io
+import boto3
+
+
+def dictionary_to_s3(data_dict: dict, s3_bucket: str, s3_folder: str, file_name: str):
+    """
+    Transforms a dictionary into a json and uploads to S3.
+    Args:
+        data_dict: dictionary with the data
+        s3_bucket: S3 bucket name where to upload the file
+        s3_folder: folder where to sotre the file within the S3 bucket
+        file_name: name of the file
+    """
+    s3_client = boto3.client("s3")
+    obj = io.BytesIO(json.dumps(data_dict).encode("utf-8"))
+    s3_client.upload_fileobj(obj, s3_bucket, f"{s3_folder}{file_name}")
```

Comment:
might be less to prone to input errors to do `os.path.join(s3_folder, file_name)`?